osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -137,11 +139,133 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
+                                                Icon = FontAwesome.fa_square,
+                                                Colour = new Color4(68,17,136,255),
+                                                Rotation = 45,
+                                            },
+                                            new TextAwesome
+                                            {
+                                                Icon = FontAwesome.fa_clock_o,
+                                                Colour = new Color4(255,221,85,255),
+                                                Scale = new Vector2(0.8f,0.8f),
+                                            }
+                                        }
+                                     },
+                                    new SpriteText
+                                    {
+                                        Margin = new MarginPadding { Left = 23},
+                                        Font = @"Exo2.0-Bold",
+                                        Colour = new Color4(255,221,85,255),
+                                        Text = ""+TimeSpan.FromMilliseconds((beatmap.Beatmap.HitObjects.Last().EndTime-beatmap.Beatmap.HitObjects.First().StartTime)).ToString(@"m\:s"),
+                                        TextSize = 17,
+                                    },
+                                    new Container
+                                    {
+                                        Margin = new MarginPadding { Left = 110, Top = 10 },
+                                        AutoSizeAxes = Axes.Both,
+                                        Children = new[] {
+                                            new TextAwesome
+                                            {
+                                                Icon = FontAwesome.fa_square,
+                                                Colour = new Color4(68,17,136,255),
+                                                Rotation = 45,
+                                            },
+                                            new TextAwesome
+                                            {
+                                                Icon = FontAwesome.fa_circle_o,
+                                                Colour = new Color4(255,221,85,255),
+                                                Scale = new Vector2(0.8f,0.8f),
+                                            }
+                                        }
+                                     },
+                                    new SpriteText
+                                    {
+                                        Margin = new MarginPadding { Left = 123},
+                                        Font = @"Exo2.0-Bold",
+                                        Colour = new Color4(255,221,85,255),
+                                        Text = ""+60000/beatmap.Beatmap.BeatLengthAt(beatmap.Beatmap.Metadata.PreviewTime)+"bpm",
```

Comment:
I feel like there should be a helper function converting beat length to BPM. It might be useful in other places, too.

Also, stable osu! displays the range of BPMs of the song, not just the BPM at preview time.